src/vmm/src/signal_handler.rs

```diff
@@ -144,6 +145,7 @@ mod tests {
 
     #[test]
     fn test_signal_handler() {
+        LOGGER.set_max_level(LevelFilter::Off);
```

Comment:
It looks a bit arbitrary that you need to use the logger and disable it for the signal_handler test to work. Can you add a comment to why is this actually needed? I assume it is because the seccomp filters.

Reply:
Done.